Node/quickstarts/callable-functions-streaming/website/index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>title</title>
+  </head>
+  <body>
+    <button id="btn-call-func">Call function</button>
+    <p id="p-result"></p>
+    <script type="module">
+      import { initializeApp } from "https://www.gstatic.com/firebasejs/11.2.0/firebase-app.js";
+      import {
+        getFunctions,
+        httpsCallable,
+        connectFunctionsEmulator,
+      } from "https://www.gstatic.com/firebasejs/11.2.0/firebase-functions.js";
+
+      const callableButton = document.getElementById("btn-call-func");
+      const resultElement = document.getElementById("p-result");
+
+      callableButton.onclick = handleClick;
+
+      // https://firebase.google.com/docs/hosting/reserved-urls#sdk_auto-configuration
+      const firebaseConfig = await fetch('/__/firebase/init.json').then(response => {
+        return response.json()
+      });
+      const app = initializeApp(firebaseConfig);
+      const functions = getFunctions(app);
+      connectFunctionsEmulator(functions, "127.0.0.1", 5001);
+      const getForecast = httpsCallable(functions, "getForecast");
+
+      const favoriteLocations = [
+            // Google HQ
+            { latitude: 37.4220199895279, longitude: -122.08531347325561 },
+            // Yosemite Valley
+            { latitude: 37.745192257741984, longitude: -119.5945133017153 },
+            // Old Faithful
+            { latitude: 44.46037818049411, longitude: -110.82802255265777 },
+          ]
+
+      async function handleClick() {
+        // reset result
+        clearUi();
+
+        const resp = await getForecast.stream({
+          locations: favoriteLocations,
+        });
+
+        // loop through the `stream` async iterable
+        for await (const forecastData of resp.stream) {
+          updateUi(forecastData);
+        }
```

Comment:
@egilmorez this sound ok for the streaming client sample code?

Reply:
I think so!  Especially with punctuation/periods in the comments. :)